build/build-mrt.yml

```diff
@@ -119,6 +119,13 @@ steps:
         }
       }
 
+- task: 333b11bd-d341-40d9-afcf-b32d5ce6f23b@2
+  displayName: 'NuGet restore MrtCore'
+  inputs:
+    restoreSolution: '${{ parameters.MRTSourcesDirectory }}\mrt\MrtCore.sln'
+    feedsToUse: config
+    nugetConfigPath:  nuget.config
```

Comment:
is this needed?

Reply:
Yes and it replaced all the previous individual restores on each project